gateway/errors.go

```diff
@@ -0,0 +1,18 @@
+package gateway
+
+import (
+	"fmt"
+	"strings"
+)
+
+// gatewayError represents an Error returned by this Middleware to a
+// requestor
+type gatewayError struct {
+	Type     string   `json:"type"`
+	Messages []string `json:"message"`
```

Comment:
According to the specs (https://github.com/manifoldco/marketplace/blob/master/specs/gateway/v1.yaml#L1069) this should just be `Message string 'json:"message"'`

Reply:
```suggestion
	Message string `json:"message"`
```